cdc/model/owner.go

```diff
@@ -259,9 +259,13 @@ func (p ProcessorsInfos) String() string {
 
 // ChangeFeedStatus stores information about a ChangeFeed
 type ChangeFeedStatus struct {
-	ResolvedTs   uint64       `json:"resolved-ts"`
-	CheckpointTs uint64       `json:"checkpoint-ts"`
-	AdminJobType AdminJobType `json:"admin-job-type"`
+	ResolvedTs   uint64 `json:"resolved-ts"`
+	CheckpointTs uint64 `json:"checkpoint-ts"`
+	// MinTableBarrierTs is the minimum table barrier timestamp of all tables.
+	// It is only used when a changefeed is started to check whether there was
+	// a table's DDL job that had not finished when the changefeed was stopped.
+	MinTableBarrierTs uint64       `json:"min-table-barrier-ts"`
```

Comment:
Why it has to be persisted to etcd?

Reply:
We need to read it from etcd when a changefeed is restarted, otherwise we can't determine whether we should start ddlPuller from `checkpointTs` or `checkpointTs-1`. Because the `status.ResolvedTs` is no longer work as a barrierTs after this PR.